src/main/java/task/Task.java

```diff
@@ -0,0 +1,98 @@
+package task;
+
+import duke.DukeException;
+import duke.Parser;
+
+public abstract class Task {
+
+    private String name;
+    private boolean done;
```

Comment:
For boolean variables it is better to start with is, has, etc.,
Overall you did a really excellent job! Your code is clean and elegant with very few violations of naming rules.